### ISSUE TITLE
Keyboard navigation doesn't work properly

### ISSUE THREAD

**Issue by VankireddyLikhithaReddy** [VankireddyLikhithaReddy/accessible-weather-shelter#78]
The keyboard navigation throughout the application is not functioning as expected.
keyboard controls like Tab, Shift + Tab, and Enter.
